src/constants/misc.js

```diff
@@ -4,6 +4,7 @@ export const NO_CATEGORY = 'overview';
 export const NO_CITY = 'all-cities';
 export const HERO_INDICATOR_ID = 'population';
 export const HERO_INDICATOR_CATEGORY = 'Planning';
+export const HERO_INDICATOR_COLOR_NAME = 'planning';
```

Comment:
I would say this is repetition, but I think the whole colour thing is a mess so...